libp2p/netMessenger.go

```diff
@@ -257,15 +258,27 @@ func setupExternalP2PLoggers() {
 	}
 }
 
-func createP2PPrivKey(seed string) (*libp2pCrypto.Secp256k1PrivateKey, error) {
-	randReader, err := randFactory.NewRandFactory(seed)
+func createP2PPrivateKey(p2pPrivateKeyBytes []byte) (libp2pCrypto.PrivKey, error) {
+	if len(p2pPrivateKeyBytes) == 0 {
+		randReader := cryptoRand.Reader
+		prvKey, err := ecdsa.GenerateKey(btcec.S256(), randReader)
+		if err != nil {
+			return nil, err
+		}
+
+		log.Info("createP2PPrivKey: generated a new private key for p2p signing")
+
```

Comment:
`createP2PPrivateKey` ? to be consistent with the rest

Reply:
done, thanks